ios/engine/KMEI/KeymanEngine/Classes/Model/InstallableLexicalModel.swift

```diff
@@ -8,6 +8,10 @@
 
 import Foundation
 
+struct InstallableConstants {
+  static let defaultVersion = "0.1.0"
```

Comment:
1.0 seems so large as a default. When someone makes a new one, is it more likely to be 1.0-ready or 0.1.0-barely ready?
With three components, maybe 0.0.1 would be better?
Or is it 1.0 because only legacy keyboards will be missing a version, and those are 1.0-ready or they  wouldn't be out there? (Of course, there aren't any 'legacy' models, yet, right?)

Reply:
Don't read too much into what a version number "means". It doesn't mean much. Versioning is mostly about understanding what is older and what is newer. Remember that semantic versioning doesn't describe readiness, but rather compatibility.

The first version available to users is 1.0. That's the versioning scheme we use for keyboards and so we'll use the same scheme for lexical models.